### PR TITLE
update master for v4.1

### PR DESCRIPTION
**Pull request by jim-minter**
```release-note
NONE
```

### CODE REVIEW

File: pkg/cluster/hash_test.go [hunk 1]
```diff
@@ -35,38 +35,35 @@ func TestHashScaleSetStability(t *testing.T) {
...
 				expectedHash: "fc64527a7a3fc02568083f200de97c9fcb22aea448e57ae46e58f1a76ba1f6f6",
 			},
 		},
-		"v4.0": {
+		"v4.1": {
 			{
 				role: api.AgentPoolProfileRoleMaster,
 				// this value should not change
-				expectedHash: "49de607e3b50aff9f2fa1a64e7a6173ab2c13969dcdc9036623ddc0666ca99af",
+				expectedHash: "9a4b049bda1bda0bf394b6e5a21c89bb8b219346261de0e6b01d2efb100f3e01",
```
Comment (ehashman) [inline pkg/cluster/hash_test.go]: Is how to generate and verify these hashes documented somewhere?

File: pkg/sync/v4/bindata.go [hunk 2]
```diff
@@ -3892,6 +3892,26 @@ func clusterrolebindingRbacAuthorizationK8sIoClusterMonitoringOperatorYaml() (*a
 	return a, nil
 }
 
+var _clusterrolebindingRbacAuthorizationK8sIoClusterMonitoringViewYaml = []byte("\x1f\x8b\x08\x00\x00\x00\x00\x00\x00\xff\x84\x8d\x41\x4a\xc4\x40\x10\x45\xf7\x7d\x8a\xba\x40\x22\xee\xa4\x77\xea\xc2\xfd\x08\xee\x6b\x3a\x7f\x32\xe5\x24\x55\x4d\x75\x75\x84\x39\xbd\x88\x41\x04\xc1\x59\x7e\x78\xff\x3d\xae\xf2\x06\x6f\x62\x9a\xc9\x8f\x5c\x46\xee\x71\x36\x97\x2b\x87\x98\x8e\x97\x87\x36\x8a\xdd\x6d\xf7\xe9\x22\x3a\x65\x7a\x5e\x7a\x0b\xf8\xc1\x16\x3c\x89\x4e\xa2\x73\x5a\x11\x3c\x71\x70\x4e\x44\xca\x2b\x32\x95\x6f\x68\x58\x4d\x25\xcc\x45\xe7\x61\x13\x7c\x24\xb7\x05\x07\x9c\xbe\x40\xae\xf2\xe2\xd6\xeb\x3f\xd1\x44\xf4\xa7\x79\x33\xd1\xfa\xf1\x1d\x25\x5a\x4e\xc3\xfe\x7e\x85\x6f\x52\xf0\x58\x8a\x75\x8d\x1f\xc1\x0c\xc5\xc6\xfb\x6c\x95\x0b\x32\x59\x85\xb6\xb3\x9c\x62\xe0\x6b\x77\xfc\x92\xa7\xcf\x00\x00\x00\xff\xff\x1b\x2e\x8a\x83\x27\x01\x00\x00")
```
Comment (ehashman) [inline pkg/sync/v4/bindata.go]: Also curious how these binary configs are generated

Comment (jim-minter) [reply pkg/sync/v4/bindata.go]: @ehashman 
go generate ./...
https://blog.golang.org/generate

File: pkg/cluster/hash_test.go (hunk 1 above)
Comment (jim-minter) [reply pkg/cluster/hash_test.go]: run the test, let it fail and output what it expects to see, copy paste from there
*cough*

**Comment by jim-minter** [comment]
flake #1481
merging because this is holding up everything else

**Pull request merged**